src/userlist/__tests__/userListSelectors-test.js

```diff
@@ -0,0 +1,63 @@
+import { fromJS } from 'immutable';
+import { sortUserList, filterUserList } from '../userListSelectors';
+
+it('empty input to filterUserList results in empty list', () => {
+  const users = fromJS([]);
+  const filteredUsers = filterUserList(users, 'some filter');
+  expect(filteredUsers).toEqual(fromJS([]));
+});
+
+it('filterUserList returns same list if no filter', () => {
+  const users = fromJS([{ email: 'user1@email.com' }, { email: 'user2@email.com' }]);
```

Comment:
Let's use example.com throughout the codebase if you don't mind. It's been established for this purpose :)